### PR TITLE
Add CKV_DOCKER_6 check for existence of MAINTAINER

### PR DESCRIPTION
**Pull request by gruebel**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Happy to see Docker support by checkov. Also happy to help here with some best or worst practice 😄 

I'm still seeing from time to time the usage of the infamous `MAINTAINER` instruction. I hope someday it will be removed completely.

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

### CODE REVIEW

File: checkov/dockerfile/checks/MaintainerExists.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        name = "Ensure that LABEL maintainer is used instead of MAINTAINER (deprecated)"
+        id = "CKV_DOCKER_6"
+        supported_instructions = ["MAINTAINER"]
+        categories = [CheckCategories.CONVENTION]
+        super().__init__(name=name, id=id, categories=categories, supported_instructions=supported_instructions)
+
+    def scan_entity_conf(self, conf):
+        for instruction, content in conf.items():
+            if instruction == "MAINTAINER":
+                return CheckResult.FAILED, conf[instruction][0]
```
Comment (tronxd) [inline checkov/dockerfile/checks/MaintainerExists.py:16]: I think the expression can be simplified into `content[0]`

**Review by tronxd — APPROVED** [review]

File: checkov/dockerfile/checks/MaintainerExists.py (hunk 1 above)
Comment (gruebel) [reply checkov/dockerfile/checks/MaintainerExists.py:16]: Yeah, you are right. Good point!

**Comment by schosterbarak** [comment]
Thanks @gruebel . Learning here, why does MAITAINER action is a bad practices?

**Comment by gruebel** [comment]
At some point Docker introduced the instruction `LABEL` which is much more flexible and if you want to add the maintainer information to the image, then you can do it via `LABEL maintainer="checkov"`. Also monitoring services like Datadog can use these labels to forward them to their platform and give you the possibility to query them.

**Comment by schosterbarak** [comment]
Thank you @gruebel

**Pull request merged**